spec/rubocop/cop/naming/accessor_method_name_spec.rb

```diff
@@ -3,81 +3,139 @@
 RSpec.describe RuboCop::Cop::Naming::AccessorMethodName, :config do
   it 'registers an offense for method get_... with no args' do
     expect_offense(<<~RUBY)
-      def get_attr
-          ^^^^^^^^ Do not prefix reader method names with `get_`.
+      def get_something
+          ^^^^^^^^^^^^^ Do not prefix reader method names with `get_`.
         # ...
       end
     RUBY
   end
 
   it 'registers an offense for singleton method get_... with no args' do
     expect_offense(<<~RUBY)
-      def self.get_attr
-               ^^^^^^^^ Do not prefix reader method names with `get_`.
+      def self.get_something
+               ^^^^^^^^^^^^^ Do not prefix reader method names with `get_`.
         # ...
       end
     RUBY
   end
 
-  it 'accepts method get_something with args' do
+  it 'accepts method get_... with args' do
```

Comment:
IMO, `get_...` looks weird, so it looks better to be consistent with `get_something`.